docs/sections/user_guide/cli/mode_config.rst

```diff
@@ -59,7 +59,94 @@ The ``uw`` mode for handling configs.
 Examples
 ~~~~~~~~
 
-TBD
+The examples that follow use YAML file ``values.yaml`` with content
```

Comment:
Good point - got it!

Reply:
While this specific case seems harmless, I wonder whether ([UW-419](https://jira.epic.oarcloud.noaa.gov/browse/UW-419) again, <sigh>) we even want to support this long-term. For example, how should we compare `config.yaml`
```
values:
  foo: 88
```
to `config.ini`
```
[values]
foo: 88
```
? In the YAML file, `foo` has an integer value; in the INI file, it's... a string? It has to be a string because **all** INI values are string, but is it _meant_ to represent an integer? Is it the same as or different from `foo` in the YAML file?

It might be best to avoid the issue for now and just do e.g. a YAML-to-YAML comparison (I wouldn't mind seeing a little more depth in the YAML tree, perhaps seeing how an entire missing _subtree_ is reported).

@christinaholtNOAA would a NML-to-NML comparison be useful?